proto/minder/v1/minder.proto

```diff
@@ -1218,7 +1218,13 @@ message ListRepositoriesResponse {
 message ReconcileEntityRegistrationRequest {
     Context context = 1;
     // entity is the entity type
-    string entity = 2;
+    string entity = 2 [
+        (buf.validate.field).string = {
+            pattern: "^[a-z]+(_[a-z]+)*$",
```

Comment:
It looks like these string enum values are capitalized?

Reply:
@evankanderson - Where is that, because I based it on these mappings here -

```go
// Entity types as string-like enums. Used in CLI and other user-facing code
const (
	// RepositoryEntity is a repository entity
	RepositoryEntity EntityType = "repository"
	// BuildEnvironmentEntity is a build environment entity
	BuildEnvironmentEntity EntityType = "build_environment"
	// ArtifactEntity is an artifact entity
	ArtifactEntity EntityType = "artifact"
	// PullRequestEntity is a pull request entity
	PullRequestEntity EntityType = "pull_request"
	// ReleaseEntity is an entity abstracting a release
	ReleaseEntity EntityType = "release"
	// PipelineRunEntity is an entity abstracting a pipeline run (eg a workflow)
	PipelineRunEntity EntityType = "pipeline_run"
	// TaskRunEntity is an entity abstracting a task run (eg a step)
	TaskRunEntity EntityType = "task_run"
	// BuildEntity is an entity that represents a software build
	BuildEntity EntityType = "build"
	// UnknownEntity is an explicitly unknown entity
	UnknownEntity EntityType = "unknown"
)
```

Reply:
Oops, I mis-read this line and the generated entity constants:

https://github.com/mindersec/minder/blob/http-send/internal/controlplane/handlers_entities.go#L45

It does look like we force these to lower-case on the server side before comparison, but I think this probably fine to start with.